rotkehlchen/data_import/importers/bitmex.py

```diff
@@ -90,26 +93,40 @@ def _consume_deposits_or_withdrawals(
         amount = deserialize_asset_amount_force_positive(csv_row['amount'])
         fee = deserialize_fee(csv_row['fee']) if csv_row['fee'] != 'null' else Fee(ZERO)
         transact_type = csv_row['transactType']
-        category = AssetMovementCategory.DEPOSIT if transact_type == 'Deposit' else AssetMovementCategory.WITHDRAWAL  # noqa: E501
+        event_type = HistoryEventType.DEPOSIT if transact_type == 'Deposit' else HistoryEventType.WITHDRAWAL  # noqa: E501
         amount = AssetAmount(satoshis_to_btc(amount))  # bitmex stores amounts in satoshis
         fee = Fee(satoshis_to_btc(fee))
         ts = deserialize_timestamp_from_date(
             date=csv_row['transactTime'],
             formatstr=timestamp_format,
             location='Bitmex Wallet History Import',
         )
-        return AssetMovement(
+        extra_data = {'memo': f'Imported from BitMEX CSV file. Transact Type: {transact_type}'}
+        if (address := deserialize_asset_movement_address(csv_row, 'address', asset)) is not None:
+            extra_data['address'] = address
+        if (transaction_id := get_key_if_has_val(csv_row, 'tx')) is not None:
+            extra_data['transaction_id'] = transaction_id
+
+        events = [AssetMovement(
+            timestamp=ts_sec_to_ms(ts),
             location=Location.BITMEX,
-            category=category,
-            address=deserialize_asset_movement_address(csv_row, 'address', asset),
-            transaction_id=get_key_if_has_val(csv_row, 'tx'),
-            timestamp=ts,
+            event_type=event_type,  # type: ignore[arg-type]  # will only be deposit or withdrawal
```

Comment:
mypy fails like this:
```
error: Argument "event_type" to "AssetMovement" has incompatible type "HistoryEventType"; expected "Literal[HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL]"  [arg-type]
```
and the `reveal_type(event_type)` simply shows its type as `HistoryEventType`.
It would seem that mypy should be able to see that it will only be a deposit or withdrawal here, but evidently it does not.

Reply:
okay so then this is not something I want you to spend more time on but probably reporting a mypy bug or checking if it's reported should be done